common/internal_model/src/test/scala/uk/ac/wellcome/models/work/generators/WorkGenerators.scala

```diff
@@ -222,6 +222,27 @@ trait WorkGenerators extends IdentifiersGenerators with InstantGenerators {
     }
   }
 
+  implicit class IndexedWorkOps(work: Work.Visible[Indexed])(
+    implicit updateState: UpdateState[Indexed]) {
+
+    def ancestors(works: Work.Visible[Indexed]*): Work.Visible[Indexed] =
+      Work.Visible[Indexed](
+        work.version,
+        work.data,
+        updateState(
+          work.state.copy(
+            relations = work.state.relations.copy(
+              ancestors = works.toList.zipWithIndex.map {
+                case (work, idx) =>
+                  Relation(work, idx + 1, 1, works.length - idx)
```

Comment:
Keyword parameters would make it easier to see what this code is doing.

Reply:
Added keyword params